test/prometheus-test.js

```diff
@@ -73,6 +73,43 @@ test('The factory function uses a custom prom-client registry', t => {
   t.end();
 });
 
+test('The add function takes an object instead of just an Array', t => {
+  t.plan(2);
+  const c1 = new CircuitBreaker(passFail, { name: 'fred' });
+  const prometheus = new PrometheusMetrics();
+  prometheus.add(c1);
+  t.equal(c1.name, 'fred');
+  t.ok(/circuit_fred_/.test(prometheus.metrics));
+  prometheus.clear();
+  t.end();
+});
+
+test('The add function provides access to metrics for all circuits', t => {
+  t.plan(6);
+  const c1 = new CircuitBreaker(passFail, { name: 'fred' });
+  const c2 = new CircuitBreaker(passFail, { name: 'bob' });
+  const c3 = new CircuitBreaker(passFail, { name: 'foo' });
+  const prometheus = new PrometheusMetrics([c1]);
+  prometheus.add([c2, c3]);
+  t.equal(c1.name, 'fred');
+  t.equal(c2.name, 'bob');
+  t.equal(c3.name, 'foo');
+  t.ok(/circuit_fred_/.test(prometheus.metrics));
+  t.ok(/circuit_bob_/.test(prometheus.metrics));
+  t.ok(/circuit_foo_/.test(prometheus.metrics));
+  prometheus.clear();
+  t.end();
+});
+
+test('The add function takes an object instead of just an Array', t => {
```

Comment:
This test description seems like a copy/paste of an earlier one.
```suggestion
test('The add function accepts zero parameters', t => {
```